### PR TITLE
Add Vietnamese concept list

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
The list was introduced in a [blog post](https://calc.hypotheses.org/7208) and it is based on IDS, so the mappings correspond to [Key-2016-1310](https://concepticon.clld.org/contributions/Key-2016-1310).

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -440,3 +440,4 @@ Blum-2024-501	Blum, Frederic and Barrientos, Carlos and Zariquiey, Roberto and L
...
 Zalizniak-2024-4583	Anna Zalizniak and Anna Smirnitskaya and Maksim Russo (Rousseau) and Ilya Gruntov and Timur Maisak and Dmitry Ganenkov and Maria Bulakh and Maria Orlova and Marina Bobrik-Fremke and Oksana Dereza and Tatiana Mikhailova and Maria Bibaeva and Mikhail Voronov	2024		4583	ranked	English	global	http://datsemshift.ru	Zalizniak2024		This is the most recent dump of the DatSemShift database, which was retrieved on February 5 (2024) from the database and then converted to CLDF. The data contains several duplicates, which we have marked by a preceding asterisk, leaving them unlinked to Concepticon and also ignoring them in the network representations.		DatSemShift
 Lastra-1986-381	Lastra de Suárez, Yolanda	1986		381		English	Spanish		Lastra1986		This concept list is the result of a study of the dialectal regions in modern Nahuatl. 	239-248	
+Tjuka-2024-1310	Tjuka, Annika	2024		1310		English	Vietnamese		Tjuka2024		The list includes 1310 concepts for Vietnamese. The concepts are based on the items provided in the [Intercontinental Dictionary Series](https://ids.clld.org/).
```
Comment (chrzyki) [inline concepticondata/conceptlists.tsv:443]: Small thing, but maybe the IDS list could be referred to with `:ref:`? (`[Intercontinental Dictionary Series](:ref:Key-2016-1310)` or similar)

**Review by chrzyki — APPROVED** [review]
Thank you!

File: concepticondata/conceptlists.tsv (hunk 1 above)
Comment (AnnikaTjuka) [reply concepticondata/conceptlists.tsv]: Sure, I updated it.

**Pull request merged**